webservices/legal_docs/load_legal_docs.py

```diff
@@ -171,6 +171,31 @@ def index_advisory_opinions():
 
         es = utils.get_elasticsearch_connection()
 
+        print("getting citations...")
+        text = db.engine.execute("""SELECT ao_no, category, ocrtext FROM aouser.document
+                                    INNER JOIN aouser.ao ON ao.ao_id = document.ao_id""")
+
+        citations = {}
+        cited_by = {}
+        for row in text:
+            print("Getting citations for %s" % row[0])
+            citations_in_doc = set()
+            text = row[2] or ''
```

Comment:
I would recommend using row['ocrtext'] - it improves the readability of the code considerably.